packages/headless/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
We should use the same tsconfig as we do for the `sit-onyx` package which extends the official @vue/tsconfig :)

Reply:
We should have a global base tsconfig, so we have consistent rules for all packages. Out-of-scope for this PR. Will create a follow-up ticket.